src/styles/createPalette.d.ts

```diff
@@ -1,50 +1,77 @@
 import { Color, Contrast } from '..';
 
+type ShadeText = {
+  primary: string;
+  secondary: string;
+  disabled: string;
+  hint: string;
+  icon: string;
+  divider: string;
+  lightDivider: string;
+};
+
+type ShadeInput = {
+  bottomLine: string;
+  helperText: string;
+  labelText: string;
+  inputText: string;
+  disabled: string;
+};
+
+type ShadeAction = {
+  active: string;
+  disabled: string;
+};
+
+type ShadeBackground = {
+  default: string;
+  paper: string;
+  appBar: string;
+  contentFrame: string;
+  status: string;
+};
+
 export type Shade = {
-  text: {
-    primary: string;
-    secondary: string;
-    disabled: string;
-    hint: string;
-    icon: string;
-    divider: string;
-    lightDivider: string;
-  };
-  input: {
-    bottomLine: string;
-    helperText: string;
-    labelText: string;
-    inputText: string;
-    disabled: string;
-  };
-  action: {
-    active: string;
-    disabled: string;
-  };
-  background: {
-    default: string;
-    paper: string;
-    appBar: string;
-    contentFrame: string;
-    status: string;
-  };
+  text: ShadeText;
+  input: ShadeInput;
+  action: ShadeAction;
+  background: ShadeBackground;
 };
 
 export const light: Shade;
 export const dark: Shade;
 
-type PaletteOptions = {
-  primary: Color;
-  secondary: Color;
-  error: Color;
-  type: Contrast;
+type CommonColors = {
```

Comment:
Please use from `colors/common`, so we do not have duplicate definition of these.

Reply:
Thanks for the remark. I added your suggested changes.